apps/client/src/components/DebugDecisionTable.js

```diff
@@ -17,9 +17,8 @@ const Answer = ({ question: { answer } }) => (
   </>
 );
 
-export default ({ checker, topic }) => {
+export default ({ checker, slug }) => {
```

Comment:
`DebugDecisionTable.js` on `develop` has this: `export default ({ checker, topic }) => {`
So I think this change can be undone?